### DISCUSSION TITLE
How to pass hash variable through ENV variables.

### DISCUSSION THREAD

**Discussion by ShubhamRasal** [fastlane/fastlane#17481] [Q&A]
upload_to_testflight(
  beta_app_review_info: {
    contact_email: "email@email.com",
    contact_first_name: "Connect",
    contact_last_name: "API",
    contact_phone: "5558675309",
    demo_account_name: "demo@email.com",
    demo_account_password: "connectapi",
    notes: "this is review note for the reviewer <3 thank you for reviewing"
  }

to pass beta_app_review_info there is one ENV variable => PILOT_BETA_APP_REVIEW_INFO
question is how I can send contact details and notes using ENV variables?

**Comment by techinpark** [comment]
Multi-line values https://github.com/bkeepers/dotenv#multi-line-values
```
If you need multiline variables, for example private keys, you can double quote strings and use the \n character for newlines:

PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----\nHkVN9…\n-----END DSA PRIVATE KEY-----\n"
```
